hazelcast-wm/pom.xml

```diff
@@ -105,6 +105,11 @@
                                     org.apache.logging.log4j;resolution:=optional,
                                     org.apache.logging.log4j.*;resolution:=optional,
                                     org.slf4j;resolution:=optional,
+                                    org.springframework.context;resolution:=optional,
+                                    org.springframework.security.core.session;version="[3.2,4)";resolution:=optional,
```

Comment:
I've added the imports as they were generated by bnd (maven-bundle-plugin), so that there is no change compared to the current manifest (if you look at the current manifest, there is also a version added to those two packages).
bnd resolves the version by looking into the manifest of the dependency and when there is a version declared on the package export, it's also applied on the import package.
Adding this version range makes sure, that the the osgi container wires at least the version, which the bundle was built against.